tensorboard/util/test_util.py

```diff
@@ -157,12 +160,10 @@ def _impl(reason=None):
 
   return _impl
 
-# TODO(#1996): Better detect TF2.0.
-_is_tf2 =  tf.__version__.startswith('2.')
 run_v1_only = _run_conditionally(
-    lambda: not _is_tf2,
+    lambda: not tensorflow_python_tf2.enabled(),
     name='run_v1_only')
 run_v2_only = _run_conditionally(
-    lambda: _is_tf2,
+    lambda: tensorflow_python_tf2.enabled(),
```

Comment:
Optional, but you don't need the lambda wrapper; it can just be `tensorflow_python_tf2.enabled`.

Reply:
Indeed, but you _do_ need the lambda wrapper above for the `not`.
Might as well be consistent, especially as the two forms are not
technically equivalent. It doesn’t matter a whole lot, in any case,
given that `run_v2_only` is never used. :-)